app/controllers/admin/rebate_forms_controller.rb

```diff
@@ -22,6 +22,8 @@ def index
 
   # GET /admin/rebate_forms/1
   def show
+    @updated_by = User.find(current_user.id)
```

Comment:
this will always say the rebate_form was updated by the current user (the user viewing the record)
If I'm logged in as Brenda, it will say it was updated by Brenda -- even though it was actually updated by Dana last week. 
You want to read this from the @rebate_form, not from who is the current logged in user.